packages/design-system-react-native/src/components/AvatarAccount/AvatarAccount.figma.tsx

```diff
@@ -0,0 +1,44 @@
+import codeConnect from '@figma/code-connect';
+import React from 'react';
+
+import { AvatarAccountSize } from '../../types';
+
+import { AvatarAccount } from './AvatarAccount';
+
+/**
+ * -- This file was auto-generated by Code Connect --
+ * React Native implementation of AvatarAccount component
+ * `props` includes a mapping from Figma properties and variants to
+ * suggested values. You should update this to match the props of your
+ * code component, and update the `example` function to return the
+ * code example you'd like to see in Figma
+ */
+
+codeConnect.connect(
+  AvatarAccount,
+  'https://www.figma.com/design/1D6tnzXqWgnUC3spaAOELN/%F0%9F%A6%8A-WIP--MMDS-Components?node-id=128%3A1673',
+  {
+    props: {
```

Comment:
More info on specific component properties here https://www.figma.com/code-connect-docs/react/#the-figma-import